src/pages/signup/Signup.module.scss

```
@@ -0,0 +1,44 @@
+.container {
+  display: flex;
+  background-color: $color-gray-fafafa;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+  padding-left: 0.75rem;
+  padding-right: 0.75rem;
```

```suggestion
  padding-left: 0.75rem;
  padding-right: 0.75rem;
```
사소하긴 하지만 이렇게 바꿔도 좋을 것 같네요!
```css
padding: 0 0.75rem
```